clients/da-vinci-client/src/main/java/com/linkedin/davinci/kafka/consumer/ActiveActiveStoreIngestionTask.java

```diff
@@ -1274,7 +1274,7 @@ protected boolean isRealTimeBufferReplayStarted(PartitionConsumptionState partit
    * used in conjunction with lag from other regions to determine ready-to-serve status.
    */
   @Override
-  protected PubSubPosition getLatestPersistedUpstreamOffsetForHybridOffsetLagMeasurement(
+  protected PubSubPosition getLatestPersistedRtPositionForLagMeasurement(
```

Comment:
Upstream could also be VT in NR phase?

Reply:
It's always RT position. For NR we've a separate method